From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 23 Nov 2023 00:55:19 +0100
Subject: gdm-pam-extensions-common: Include string header

We do various string operations in the header, so include it
implicitly.
---
 pam-extensions/gdm-pam-extensions-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pam-extensions/gdm-pam-extensions-common.h b/pam-extensions/gdm-pam-extensions-common.h
index 5201823..16bee5c 100644
--- a/pam-extensions/gdm-pam-extensions-common.h
+++ b/pam-extensions/gdm-pam-extensions-common.h
@@ -25,6 +25,7 @@
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <string.h>
 #include <limits.h>
 
 #include <security/pam_appl.h>
